### ISSUE TITLE
When a git commit is not (yet) available, Viash creates a faulty `Dockerfile`

### ISSUE THREAD

**Issue by tverbeiren** [viash-io/viash#210]
When no initial git commit is made yet, the following config just based on a base image:

```yaml
...
platforms:
  - type: docker
    image: ubuntu:20.04
```

Generates a `Dockerfile` like this:

```
...
LABEL org.opencontainers.image.revision="HEAD
fatal: ambiguous argument 'HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'"
...
```

**Comment by rcannood** [comment]
Observed results:
```bash
$ mkdir foo
$ cp -r src/test/resources/testr foo
$ cd foo
$ git init
$ viash run testr/script.vsh.R -p docker -- ---dockerfile
FROM rocker/tidyverse:latest

RUN apt-get update && \
  DEBIAN_FRONTEND=noninteractive apt-get install -y libhdf5-serial-dev && \
  rm -rf /var/lib/apt/lists/*

RUN Rscript -e 'if (!requireNamespace("remotes", quietly = TRUE)) install.packages("remotes")' && \
  Rscript -e 'if (!requireNamespace("BiocManager", quietly = TRUE)) install.packages("BiocManager")' && \
  Rscript -e 'if (!requireNamespace("limma", quietly = TRUE)) BiocManager::install("limma")' && \
  Rscript -e 'remotes::install_cran(c("optparse"), repos = "https://cran.rstudio.com")' && \
  Rscript -e 'remotes::install_github(c("dynverse/dynutils@devel"), repos = "https://cran.rstudio.com")'

LABEL org.opencontainers.image.description="Companion container for running component testr"
LABEL org.opencontainers.image.created="2022-09-05T17:03:06+02:00"
LABEL org.opencontainers.image.revision="HEAD
fatal: ambiguous argument 'HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'"
```

Expected output:

```bash
$ mkdir foo
$ cp -r src/test/resources/testr foo
$ cd foo
$ git init
$ viash run testr/script.vsh.R -p docker -- ---dockerfile
FROM rocker/tidyverse:latest

RUN apt-get update && \
  DEBIAN_FRONTEND=noninteractive apt-get install -y libhdf5-serial-dev && \
  rm -rf /var/lib/apt/lists/*

RUN Rscript -e 'if (!requireNamespace("remotes", quietly = TRUE)) install.packages("remotes")' && \
  Rscript -e 'if (!requireNamespace("BiocManager", quietly = TRUE)) install.packages("BiocManager")' && \
  Rscript -e 'if (!requireNamespace("limma", quietly = TRUE)) BiocManager::install("limma")' && \
  Rscript -e 'remotes::install_cran(c("optparse"), repos = "https://cran.rstudio.com")' && \
  Rscript -e 'remotes::install_github(c("dynverse/dynutils@devel"), repos = "https://cran.rstudio.com")'

LABEL org.opencontainers.image.description="Companion container for running component testr"
LABEL org.opencontainers.image.created="2022-09-05T17:01:05+02:00"
```

Branch `bugfix/docker_annotations` should solve this issue, still need to add a test.

**Comment by Grifs** [comment]
Fixed in Viash 0.6.0
